lib/middleware/request-handler.ts

```diff
@@ -212,9 +171,9 @@ export function onSubscribe(req: Interface.IOurRequest,
         }
 
         req.body.forEach((s: {'correlationId': number;
-                              'security': string;
-                              'fields': string[];
-                              'options'?: any }): void => {
+            'security': string;
```

Comment:
Actually, maybe a type alias may be more appropriate.  I meant the indent of `correlationId` on a new line to be in conformance with the object literal style guide, but that seems awkward here the more I think about.